src/services/event.js

```diff
@@ -5,98 +5,124 @@
+};
+
+/**
+ * Get chapel events attended by the user during the current term
+ * @return {AttendedEvent[]} all CL&W events attended by the user, formatted and sorted
+ */
+const getAttendedChapelEvents = async () => {
+  const termCode = session.getTermCode();
+  let attendedEvents = await http.get(`events/chapel/${termCode}`);
+  return attendedEvents.map((e) => formatevent(e)).sort(sortAtndEventsByTime);
+};
 
+/**
+ *  Format an event for display on the front end
+ * @param {Event} event The event to format
+ * @returns {Event} The formatted event
+ */
 function formatevent(event) {
-  let beginTime;
-  let endTime;
-  if (event.Occurrences[0] && event.Occurrences[0][0] && event.Occurrences[0][1]) {
-    beginTime = DateTime.fromISO(event.Occurrences[0][0]).toFormat('t');
-    endTime = DateTime.fromISO(event.Occurrences[0][1]).toFormat('t');
+  let formattedEvent = { ...event };
+  if (event.Occurrences[0]) {
+    let beginTime = DateTime.fromISO(event.Occurrences[0].StartDate).toFormat('t');
+    let endTime = DateTime.fromISO(event.Occurrences[0].EndDate).toFormat('t');
+    formattedEvent.timeRange = `${beginTime} - ${endTime}`;
+    formattedEvent.date = DateTime.fromISO(event.Occurrences[0].StartDate).toFormat('LLL d, yyyy');
   }
-  const timeRange = `${beginTime} - ${endTime}`;
-  event.timeRange = timeRange;
 
-  let date;
-  if (event.Occurrences[0] && event.Occurrences[0][0]) {
-    date = DateTime.fromISO(event.Occurrences[0][0]).toFormat('LLL d, yyyy');
-  }
-  event.date = date;
+  formattedEvent.title = event.Event_Title || event.Event_Name;
 
-  let title;
-  if (event.Event_Title === '') {
-    title = event.Event_Name;
-  } else {
-    title = event.Event_Title;
-  }
-  event.title = title;
+  formattedEvent.location = event.Occurrences[0].Location || 'No Location Listed';
 
-  let location;
-  if (event.Occurrences[0] && event.Occurrences[0][2]) {
-    location = `${event.Occurrences[0][2]} `;
-  } else {
-    location = 'No location Listed';
+  if (!formattedEvent.Description) {
+    formattedEvent.Description = 'No description available';
   }
-  event.location = location;
 
-  if (event.Description) {
-    if (event.Description === '' || event.Description.substring(0, 4) === '<res') {
-      event.Description = 'No description available';
-    }
-    event.Description = event.Description.replace(/&(#[0-9]+|[a-zA-Z]+);/g, ' ').replace(
-      /<\/?[^>]+(>|$)/g,
-      ' ',
-    );
-  }
-  return event;
+  // Remove markup from event description.
+  formattedEvent.Description = formattedEvent.Description.replace(
+    /&(#[0-9]+|[a-zA-Z]+);/g,
+    ' ',
+  ).replace(/<\/?[^>]+(>|$)/g, ' ');
+
+  return formattedEvent;
 }
+
 function filterbyCategory(filters, allEvents) {
   let filteredEvents = [];
   if (
     filters.chapelOffice ||
     filters.art ||
     filters.cec ||
-    filters.calendar ||
     filters.admissions ||
     filters.sports ||
     filters.studentLife ||
-    filters.fair ||
-    filters.academics ||
```

Comment:
It's certainly not ideal, but unfortunately we have no good means of replacing them, since they relied on event types that no longer exist. It might be possible to build something that looked for certain organizations, but it would introduce a lot of added complexity without ever being as reliable as the event types were. Additionally, I'm not positive that these filters were widely used when they did work.